paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewAction.kt

```diff
@@ -10,7 +10,7 @@ internal sealed class CustomerSheetViewAction {
     object OnEditPressed : CustomerSheetViewAction()
     object OnAddCardPressed : CustomerSheetViewAction()
     object OnPrimaryButtonPressed : CustomerSheetViewAction()
-    class OnFormValuesChanged(val formFieldValues: FormFieldValues?) : CustomerSheetViewAction()
+    class OnFormDataUpdated(val formData: FormViewModel.ViewData) : CustomerSheetViewAction()
```

Comment:
That is true, we aren't going through the view anymore, but I do wonder if we still need this, since it is the public entry point for view model actions for testing. Otherwise, it makes testing this harder.